sql/core/src/test/resources/sql-tests/results/postgreSQL/float8.sql.out

```diff
@@ -490,191 +490,191 @@ struct<five:string,f1:double>
 -- !query
 SELECT sinh(double('1'))
 -- !query schema
-struct<SINH(CAST(1 AS DOUBLE)):double>
+struct<sinh(CAST(1 AS DOUBLE)):double>
 -- !query output
 1.1752011936438014
 
 
 -- !query
 SELECT cosh(double('1'))
 -- !query schema
-struct<COSH(CAST(1 AS DOUBLE)):double>
+struct<cosh(CAST(1 AS DOUBLE)):double>
 -- !query output
 1.543080634815244
 
 
 -- !query
 SELECT tanh(double('1'))
 -- !query schema
-struct<TANH(CAST(1 AS DOUBLE)):double>
+struct<tanh(CAST(1 AS DOUBLE)):double>
 -- !query output
 0.7615941559557649
 
 
 -- !query
 SELECT asinh(double('1'))
 -- !query schema
-struct<ASINH(CAST(1 AS DOUBLE)):double>
+struct<asinh(CAST(1 AS DOUBLE)):double>
 -- !query output
 0.8813735870195429
 
 
 -- !query
 SELECT acosh(double('2'))
 -- !query schema
-struct<ACOSH(CAST(2 AS DOUBLE)):double>
+struct<acosh(CAST(2 AS DOUBLE)):double>
 -- !query output
 1.3169578969248166
 
 
 -- !query
 SELECT atanh(double('0.5'))
 -- !query schema
-struct<ATANH(CAST(0.5 AS DOUBLE)):double>
+struct<atanh(CAST(0.5 AS DOUBLE)):double>
 -- !query output
 0.5493061443340548
 
 
 -- !query
 SELECT sinh(double('Infinity'))
 -- !query schema
-struct<SINH(CAST(Infinity AS DOUBLE)):double>
+struct<sinh(CAST(Infinity AS DOUBLE)):double>
 -- !query output
 Infinity
 
 
 -- !query
 SELECT sinh(double('-Infinity'))
 -- !query schema
-struct<SINH(CAST(-Infinity AS DOUBLE)):double>
+struct<sinh(CAST(-Infinity AS DOUBLE)):double>
 -- !query output
 -Infinity
 
 
 -- !query
 SELECT sinh(double('NaN'))
 -- !query schema
-struct<SINH(CAST(NaN AS DOUBLE)):double>
+struct<sinh(CAST(NaN AS DOUBLE)):double>
 -- !query output
 NaN
 
 
 -- !query
 SELECT cosh(double('Infinity'))
 -- !query schema
-struct<COSH(CAST(Infinity AS DOUBLE)):double>
+struct<cosh(CAST(Infinity AS DOUBLE)):double>
```

Comment:
Can we keep this unchanged?

Reply:
Yup, I think that will scope this PR smaller, and I don't have to fix R tests as well.